src/dotnet-interactive-vscode/src/tests/unit/ipynbMetadata.test.ts

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { expect } from 'chai';
+import { DotNetCellMetadata, getCellLanguage, getDotNetMetadata, getLanguageInfoMetadata, LanguageInfoMetadata, withDotNetMetadata } from '../../ipynbUtilities';
+
+describe('ipynb metadata tests', () => {
+
+    //------------------------------------------------------- document metadata
```

Comment:
We can do nested `describe`s instead of comments for categorization, which will help make test output clearer.

Reply:
done.